### PR TITLE
Fix win

### PR DESCRIPTION
**Pull request by brian-rose**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by brian-rose** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,28 +10,31 @@ source:
...
-  number: 1
-  skip: true  # [win and py27]
+  number: 2
+  skip: true  # [win and py<30]
 
 requirements:
   build:
-    - {{ compiler('fortran') }}
+    - {{ compiler('fortran') }}  # [not win]
+    - flang                      # [win]
```
Comment (jhkennedy) [inline recipe/meta.yaml]:
```suggestion
    - {{ compiler('fortran') }}
    - {{ compiler('m2w64_fortran') }} # [win]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,28 +10,31 @@ source:
...
     - {{ compiler('c') }}
   host:
-    - flang  # [win and py3k]
+    - flang  # [win]
```
Comment (jhkennedy) [inline recipe/meta.yaml]:
```suggestion
    - m2w64-toolchain  # [win]
    - libpython  # [win]
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -10,28 +10,31 @@ source:
...
     - python
     - setuptools
     - pip
     - numpy
+    - nomkl          # [win]
   run:
     - python
     - {{ pin_compatible('numpy') }}
-    - libflang  # [win and py3k]
+    - libflang    # [win]
```
Comment (jhkennedy) [inline recipe/meta.yaml]:
```suggestion
```
According to conda-forge docs, this should be automatically added to the runtime requirements, I think
https://conda-forge.org/docs/maintainer/knowledge_base.html#compilers

**Review by jhkennedy — COMMENTED** [review]
@brian-rose I'm a little out of my element with window+fortran, but I've had luck w/ f2py on windows by making these changes. Not sure if it'll work here though...

File: recipe/meta.yaml (hunk 1 above)
Comment (jhkennedy) [reply recipe/meta.yaml]: you may have to also revert the changes to `build.bat`

File: recipe/meta.yaml (hunk 3 above)
Comment (jhkennedy) [reply recipe/meta.yaml]: Well, maybe for flang it's needed. Looking closer at https://github.com/conda-forge/fastscapelib-f2py-feedstock/pull/12 which switched to flag

File: recipe/meta.yaml (hunk 1 above)
Comment (jhkennedy) [reply recipe/meta.yaml]: may also need `{{ compiler('m2w64_c') }}`

**Comment by brian-rose** [comment]
@conda-forge-admin, please rerender

**Comment by brian-rose** [comment]
@jhkennedy I'm definitely out of my league too! Changing the Windows recipe is always a shot in the dark for me. But happy to try your suggestions. Thanks!

**Comment by brian-rose** [comment]
Looking back on old commits, it seems that I switched from ``{{ compiler('m2w64_fortran') }}`` to `flang` back in December 2018: https://github.com/conda-forge/climlab-feedstock/commit/c3e1f4f1dda7d32231efdcf9cc8b414418524ed4#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9a.

**Comment by brian-rose** [comment]
Success! Windows builds are passing.

I'm going to try some cleanup before merging this.

**Comment by jhkennedy** [comment]
:partying_face:

**Comment by brian-rose** [comment]
Merging. The failure on linux pypy we can look at elsewhere.

**Pull request merged**